qa/restart-upgrade/src/test/resources/processor/IndexMappings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "knn.algo_param.ef_search": 100,
+      "refresh_interval": "30s",
+      "default_pipeline": "%s"
+    },
+    "number_of_shards": 1,
```

Comment:
Good point. I will update it.

Reply:
1 shard use-case is a valid use case for hybrid search but not for processors. I would say lets go with 3 shards and 1 replica. This will ensure that we are handling all the valid cases for text embedding processor. For Hybrid Search we should create test for different shard configurations.